.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 npm*.log
 README.html
+.eslint.log
```

Comment:
This should stay tracked. When you run the linter (`npm run lint`) the the log will stay blank if there were no errors, otherwise it will have warnings that need to be fixed.